From fb884394524480b64414cca1046a311ed409b91d Mon Sep 17 00:00:00 2001
From: Paul Keith <javelinanddart@gmail.com>
Date: Thu, 7 Feb 2019 11:54:48 -0600
Subject: [PATCH 11/15] common: Label livedisplay 2.0 sysfs service

* Add label for acl node too for devices that use it

Change-Id: Iaa69ad07f122a6a74182afe7db49d4c21bd88842

# Conflicts:
#	common/private/genfs_contexts
---
 common/vendor/file_contexts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/common/vendor/file_contexts b/common/vendor/file_contexts
index 89b5264..508c63f 100644
--- a/common/vendor/file_contexts
+++ b/common/vendor/file_contexts
@@ -7,6 +7,7 @@
 # LiveDisplay HAL
 /(vendor|system/vendor)/bin/hw/vendor\.lineage\.livedisplay@2\.0-service-legacymm u:object_r:hal_lineage_livedisplay_default_exec:s0
 /(vendor|system/vendor)/bin/hw/vendor\.lineage\.livedisplay@2\.0-service-sdm u:object_r:hal_lineage_livedisplay_default_exec:s0
+/(vendor|system/vendor)/bin/hw/vendor\.lineage\.livedisplay@2\.0-service-sysfs u:object_r:hal_lineage_livedisplay_default_exec:s0
 
 # Trust HAL
 /(vendor|system/vendor)/bin/hw/vendor\.lineage\.trust@1\.0-service u:object_r:hal_lineage_trust_default_exec:s0
-- 
2.17.1

